### PR TITLE
feat(github): backend for open PR comments

### PR DESCRIPTION
**Pull request by mifu67**
Add org option to enable/disable the Github Open PR Comment Bot. Separating toggle PR into FE and BE.

### CODE REVIEW

**Review by EricHasegawa — COMMENTED** [review]
LGTM - will let Cathy or someone else with more context approve. If you're blocked by this ping me and I can approve!

File: src/sentry/api/serializers/models/organization.py
Comment (cathteng) [inline src/sentry/api/serializers/models/organization.py:45]: let's condense the `GITHUB_OPEN_PR_BOT_DEFAULT` and `GITHUB_PR_BOT_DEFAULT` into a single `COMMENT_BOT_DEFAULT` that's always true

**Review by cathteng — APPROVED** [review]
looks good, but just one thing before you merge

**Pull request merged**
